lib/services/sku-pack-service.js

```diff
@@ -1,10 +1,13 @@
-// Copyright 2015, EMC, Inc.
+// Copyright 2016, EMC, Inc.
```

Comment:
If you just change the file (not creating a brand-new file), you don't need to change the copyright.